packages/streamline/src/theme/shadows.ts

```diff
@@ -27,6 +27,10 @@ export const shadows = {
     },
     shadowRadius: 20,
     shadowOpacity: 0.12,
-    elevation: 4,
+    elevation: 12,
```

Comment:
keep 4?

Reply:
It looks like the figma with 12... We need to align with IAN